staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -712,7 +712,7 @@ func expectConversionFailureMessage(id, message string) func(t *testing.T, ctc *
 			})
 		}
 		for _, subresource := range []string{"status", "scale"} {
-			for _, verb := range []string{"get", "udpate", "patch"} {
+			for _, verb := range []string{"get", "update", "patch"} {
 				t.Run(fmt.Sprintf("%s-%s", subresource, verb), func(t *testing.T) {
 					switch verb {
 					case "create":
```

Comment:
same thing here... passing a v1beta1 object to a v1beta2 endpoint does not look right